mmdt/templates/base.html

```diff
@@ -73,6 +73,8 @@ <h2>Get in touch</h2>
 <script src="{% static 'assets/js/breakpoints.min.js'%}"></script>
 <script src="{% static 'assets/js/util.js' %}"></script>
 <script src="{% static 'assets/js/main.js' %}"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
```

Comment:
please download this and use as offline. We better not to use `cdn`

Reply:
Hote ama.